html/pfappserver/lib/pfappserver/Form/Config/AdminRoles.pm

```diff
@@ -90,7 +90,7 @@ sub options_actions {
     my @options;
 
     map {
```

Comment:
Please use a foreach instead of map.
It's easier to read and less prone to breaking when we later modify the code.

Or better yet, stick it into a subroutine. See PBP 6.13.